src/app/pages/questions/components/question/text-input/text-input.component.ts

```diff
@@ -85,9 +86,9 @@ export class TextInputComponent implements OnInit {
 
   initDuration() {
     const minutes = this.addLeadingZero(Array.from(Array(60).keys()))
-    if (this.showSeconds) this.timePickerValues.push(minutes)
     const longHours = this.addLeadingZero(Array.from(Array(24).keys()))
     this.durationPickerValues = [longHours, minutes]
+    this.defaultDurationPickerValue = ["00","00"]
```

Comment:
Maybe this is not needed since this will default to the first values if not specified?

Reply:
This was just one way to fix the problem of undefined values if you don't change the wheel selector. Also, if this is not set, the wheel selector does not have a `selection` object, and the timeout function in the value change handler throws an exception, effectively locking the selector forever...